mmdet/datasets/pipelines/loading.py

```diff
@@ -76,6 +76,117 @@ def __repr__(self):
+        img = mmcv.imfrombytes(img_bytes, flag=self.color_type)
+        if self.to_float32:
+            img = img.astype(np.float32)
+
+        # e.g. img_prefix = '' or 'ref_'
+        img_prefix = img_info_key[-12:-8]
+        results[f'{img_prefix}filename_{str(fname_id)}'] = fname
+        results[f'{img_prefix}ori_filename_{str(fname_id)}'] = ori_fname
+        results[f'{img_prefix}img_{str(fname_id)}'] = img
+        results['img_fields'].append(f'{img_prefix}img_{str(fname_id)}')
+
+        results['img_shape'] = img.shape
+        results['ori_shape'] = img.shape
+        # Set initial values for default meta_keys
+        results['pad_shape'] = img.shape
+        results['scale_factor'] = 1.0
+        num_channels = 1 if len(img.shape) < 3 else img.shape[2]
+        results['img_norm_cfg'] = dict(
+            mean=np.zeros(num_channels, dtype=np.float32),
+            std=np.ones(num_channels, dtype=np.float32),
+            to_rgb=False)
+        return results
+
+    def __call__(self, results):
+        if self.file_client is None:
+            self.file_client = mmcv.FileClient(**self.file_client_args)
+
+        error_flag = 1
+        for img_info_key in self.img_info_keys:
+            if img_info_key in results:
+                error_flag = 0
+        if error_flag:
+            raise KeyError(f'"results" should contain at least one key in '
+                           f'{self.img_info_keys}, obtain {results.keys()}')
+
+        results['img_fields'] = []
+        for img_info_key in self.img_info_keys:
+            if img_info_key not in results:
+                continue
+            ori_filenames = results[img_info_key]['filename']
+            if type(ori_filenames) is str:
+                if results['img_prefix'] is not None:
+                    filenames = [
+                        osp.join(results['img_prefix'], ori_filenames)
+                    ]
+                else:
+                    filenames = [ori_filenames]
+                ori_filenames = [ori_filenames]
+            elif type(ori_filenames) is list:
+                if results['img_prefix'] is not None:
+                    filenames = [
+                        osp.join(results['img_prefix'], fname)
+                        for fname in ori_filenames
+                    ]
+                else:
+                    filenames = ori_filenames
+            else:
+                raise TypeError('the type of filename must be a str or list')
+
+            for fname_id, fname in enumerate(filenames):
+                ori_fname = ori_filenames[fname_id]
+                results = self.load_img_core(results, fname_id, fname,
+                                             ori_fname, img_info_key)
+
+        return results
+
+    def __repr__(self):
+        repr_str = self.__class__.__name__
+        repr_str += f'(\nimg_info_keys={self.img_info_keys},\n'
+        repr_str += f'to_float32={self.to_float32},\n'
```

Comment:
f'    to_float32={self.to_float32},\n', there should be four spaces here.

Reply:
Done